### PR TITLE
Delete command filter

### PR DESCRIPTION
**Pull request by hugoj-goncalves**
Delete does not have a command filter like Update and Insert methods, would be cool to have also.
I'm not sure if those are the best places to call the filter

### CODE REVIEW

File: src/ServiceStack.OrmLite/OrmLiteWriteExpressionsApi.cs [hunk 1]
```diff
@@ -246,7 +246,7 @@ public static int Delete<T>(this IDbConnection dbConn, Expression<Func<T, bool>>
         ///   db.Delete&lt;Person&gt;(q.Where(p => p.Age == 27));
         ///   DELETE FROM "Person" WHERE ("Age" = 27)
         /// </summary>
-        public static int Delete<T>(this IDbConnection dbConn, SqlExpression<T> where)
+        public static int Delete<T>(this IDbConnection dbConn, SqlExpression<T> where, Action<IDbCommand> commandFilter = null)
         {
             return dbConn.Exec(dbCmd => dbCmd.Delete(where));
```
Comment (madaleno) [inline src/ServiceStack.OrmLite/OrmLiteWriteExpressionsApi.cs:251]: shouldn't this be: ... dbCmd.Delete(where, commandFilter)); ?

Comment (mythz) [reply src/ServiceStack.OrmLite/OrmLiteWriteExpressionsApi.cs:251]: Yep it should be, thx for catching it! I'll resolve it on my side.

**Comment by mythz** [comment]
FYI made minor tweaks and added equivalent support for Delete Async APIs in [this commit](https://github.com/ServiceStack/ServiceStack.OrmLite/commit/e818b16722b369742cc3986f07b83df3d730848a).

This needed these changes are now [available on MyGet](https://docs.servicestack.net/myget).

File: src/ServiceStack.OrmLite/OrmLiteWriteExpressionsApi.cs (hunk 1 above)
Comment (hugoj-goncalves) [reply src/ServiceStack.OrmLite/OrmLiteWriteExpressionsApi.cs:251]: Ops

**Pull request merged**